src/index/ExternalVocabulary.cpp

```diff
@@ -15,10 +15,10 @@
 template <class Comp>
 string ExternalVocabulary<Comp>::operator[](Id id) const {
   off_t ft[2];
+  off_t at = _startOfOffsets + id * sizeof(off_t);
+  _file.read(ft, sizeof(ft), at);
```

Comment:
I see what5 you meant by "confusing":
One asks themselves "Why is this `assert` of something uninitialized not undefined",
Although we write to `from` since we write to `ft`.